app/notebook-store.js

```diff
@@ -92,6 +93,11 @@ function _getFilename (req) {
     return name;
 }
 
+function _hiddenFileFilter (filename) {
+    //check for hidden files or directories, returns true if file is not hidden
+    return !(/(^|\/)\.[^\/\.]/g).test(filename);
```

Comment:
Yeah, I'd do the minimum for simplicity. A more complex solution can be used in the future if the need arises.